src/btree/bt_split.c

```diff
@@ -1030,6 +1030,16 @@ __split_parent(WT_SESSION_IMPL *session, WT_REF *ref,
 #endif
 
 	/*
+	 * Page-delete information is only read when the WT_REF state is
+	 * WT_REF_DELETED.  The page-delete memory wasn't added to the
+	 * parent's footprint, ignore it here.
+	 */
+	if (ref->page_del != NULL) {
```

Comment:
@michaelcahill This is the only part of the back port I'm unsure about. On develop this free and the subsequent ``WT_PUBLISH`` are wrapped in an ``if (discard)`` which would be equivalent to ``if (closing)`` on this branch.

I know that's because split_parent is used in different ways on develop, so I *think* this is the right approach, but wanted to call it out to make sure you agree.